src/components/New/Filter/Filter.tsx

```diff
@@ -0,0 +1,65 @@
+import { Popover, Transition } from "@headlessui/react";
+import { PropsWithChildren } from "react";
+import FilterIcon from "src/assets/icons/FilterIcon";
+import Button, { ButtonSize, ButtonType } from "src/components/Button";
+import { useIntl } from "src/hooks/useIntl";
+import Refresh from "src/icons/Refresh";
+import { cn } from "src/utils/cn";
+
+export function Filter({
+  children,
+  isActive,
+  onClear,
+}: PropsWithChildren<{
+  isActive: boolean;
+  onClear: () => void;
+}>) {
+  const { T } = useIntl();
+
+  return (
+    <Popover className="relative">
+      {({ open }) => (
+        <>
+          <Popover.Button
+            as={Button}
+            type={ButtonType.Secondary}
+            size={ButtonSize.Sm}
+            pressed={open}
+            className={cn({
+              "border-spacePurple-200 text-spacePurple-100": isActive,
+            })}
+          >
+            <FilterIcon /> {T("filter.title")}
+          </Popover.Button>
+
+          <Transition
+            enter="transition duration-100 ease-out"
+            enterFrom="transform scale-95 opacity-0"
+            enterTo="transform scale-100 opacity-100"
+            leave="transition duration-75 ease-out"
+            leaveFrom="transform scale-100 opacity-100"
+            leaveTo="transform scale-95 opacity-0"
```

Comment:
I think you can apply transitions and transformations solely on `enter` and `leave` props

```suggestion
          <Transition
            enter="transform transition duration-100 ease-out"
            enterFrom="scale-95 opacity-0"
            enterTo="scale-100 opacity-100"
            leave="transform transition duration-75 ease-out"
            leaveFrom="scale-100 opacity-100"
            leaveTo="scale-95 opacity-0"
```

Reply:
c407eabe5c238ed9782920faa5093f269bc6317b